drape_frontend/visual_params.hpp

```diff
@@ -98,7 +98,13 @@ m2::RectD GetRectForDrawScale(double drawScale, m2::PointD const & center);
 uint32_t CalculateTileSize(uint32_t screenWidth, uint32_t screenHeight);
 
 void ExtractZoomFactors(ScreenBase const & s, double & zoom, int & index, float & lerpCoef);
-float InterpolateByZoomLevels(int index, float lerpCoef, std::vector<float> const & values);
+float InterpolateByZoomLevels(int index, float lerpCoef, float const * values,
+                              float const * valuesEnd);
```

Comment:
- rename InterpolateByZoomLevelsImpl
- accept size instead of range's end

Reply:
Done